app/src/main/java/com/sun/spoonacular/utils/ImageExt.kt

```diff
@@ -0,0 +1,11 @@
+package com.sun.spoonacular.utils
+
+import android.content.Context
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+
+fun ImageView.loadUrl(context: Context, imageApi: String) {
+    Glide.with(context)
+        .load(imageApi)
+        .into(this)
```

Comment:
nên thêm các trường hợp load lỗi, placeholder để tăng UX 
```suggestion
        .load(imageApi)
        .error()
        .placeHolder()
        .into(this)
```